students/brandon_nguyen/session03/strformat_lab.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+from decimal import Decimal
+
+#!/usr/bin/env python3
+#Week3 Excercise string formating
+#Student: Brandon Nguyen - Au2018
+
+#task 1 and 2
+
+def strTask12():
+    strT=( 2, 123.4567, 10000, 12345.67)
+    #formt to produce 'file_002 :   123.46, 1.00e+04, 1.23e+04'
+    print("\'file_00{}:   {:.2f}, {:.2e}, {:.2e}\'".format(strT[0],strT[1],Decimal(strT[2]),Decimal(strT[3]))) #replace return/print to test
```

Comment:
If `strT[0]` is a number with more than 1 digit (e.g., 11), your print out will be `file_0011: ...`, which is not what was asked.

You used `{:02d}` below in line 25, and you should use that here as well, to put 0 padding on the left with the fixed size width.